ui/hooks/src/use-comments.new.ts

```diff
@@ -170,13 +170,13 @@ export function useCreateComment() {
 }
 
 // hook used for editing a comment
-export function useEditComment(commentID: string, hasCommentData: boolean) {
+export async function useEditComment(commentID: string, hasCommentData: boolean) {
   const sdk = getSDK();
   const queryClient = useQueryClient();
 
   return useMutation(
     (comment: IPublishData & { postID: string }) => {
-      if (!hasCommentData) return;
+      if (!hasCommentData) return Promise.resolve(null);
```

Comment:
You can just use Promise.resolve() which will resolve to `undefined` value like `return;`